aptos-move/framework/src/natives/aggregator_natives/aggregator.rs

```diff
@@ -195,6 +258,15 @@ pub fn make_all(
     features: Arc<Features>,
 ) -> impl Iterator<Item = (String, NativeFunction)> {
     let natives = [
+        (
+            "try_add",
```

Comment:
@vusirikala actually you can do the following (and I think this will be ok from compatibility point of view): keep `add` and `sub` in the native table (so all gas parameters stay the same) but instead use `native_try_add` instead of `add` for example. This way we swap implementation and can drop old natives :)

Still talk to @vgao1996 if we can do this nicer

Reply:
Yeah. I'm actually using the same gas parameters for both `add` and `try_add` in the current code.